css/values.css

```diff
@@ -0,0 +1,557 @@
+	.desktop-parent {
+		display: flex;
+		flex-direction: column;
+		gap: 20px;
+		margin: 190px auto 193px auto;
+		max-width: 1328px;
+	}
+	.values-main {
+		display: flex;
+		position: absolute;
+		left: 50%;
+		top: 45%;
+	}
+	.values-card {
+		margin: 0px 10px;
+	}
+	.copperback,
+	.authenticity,
+	.education {
+		margin-right: 10px;
+	}
+	.sustainability-text,
+	.grapeback,
+	.technology {
+		margin-left: 10px;
+	}
+	.values-card h2 {
+		margin: 20px 0px 0px 20px;
+		font-size: var(--font36);
+		font-weight: var(--font-weight-bold);
+		line-height: var(--line-height40);
+		text-align: left;
+	}
+	.values-card p {
+		display: flex;
+		font-size: var(--font21);
+		font-weight: var(--font-weight-regular);
+		line-height: var(--line-height26);
+		margin: var(--spacing20);
+	}
+
+	.onyxback,
+	.grapeback,
+	.tealback {
+		padding-right: 44px;
+	}
+
+	/* Cards need to change from column to row with some stylized sizing and padding*/
+	.desktop-group {
+		display: flex;
+		flex-direction: row;
+		height: 290px;
+	}
+	/* Gradients */
+	.hero {
+		width: 100vw;
+		height: 100vh;
+		background-image: linear-gradient(
+				260deg,
+				rgba(9, 9, 9, 0.8) 46.9%,
+				rgba(9, 9, 9, 0) 86.82%
+			),
+			url(../assets/images/pages/values/hero-values.jpg);
+		background-position: center center;
+		background-size: cover;
+		background-repeat: no-repeat;
+		position: relative;
+	}
+	.hero img {
+		display: none;
+	}
+	.sustainability {
+		/* background-color: var(--color-onyx); */
+		background-image: linear-gradient(
+				269deg,
+				#0e0e0e 46.26%,
+				rgba(27, 27, 27, 0) 99.04%
+			),
+			url(../assets/images/pages/values/sustainability.jpg);
+		background-position: center center;
+		background-size: cover;
+		background-repeat: no-repeat;
+	}
+	.authenticity {
+		background-image: linear-gradient(
+				88deg,
+				#b91515 47.26%,
+				rgba(185, 21, 21, 0.73) 55.57%,
+				rgba(119, 45, 45, 0.4) 68.47%
+			),
+			url(../assets/images/pages/values/authenticity.jpg);
+		background-position: center center;
+		background-size: cover;
+		background-repeat: no-repeat;
+	}
+	.education {
+		background-image: linear-gradient(
+				90deg,
+				#184f85 63.23%,
+				rgba(24, 79, 133, 0) 83.39%
+			),
+			url(../assets/images/pages/values/education.jpg);
+		background-position: center center;
+		background-size: cover;
+		background-repeat: no-repeat;
+	}
+
+	.technology {
+		background-image: linear-gradient(
+				280deg,
+				#007a80 67.16%,
+				rgba(0, 122, 128, 0) 80.36%
+			),
+			url(../assets/images/pages/values/techinnovation.jpg);
+		background-position: center center;
+		background-size: cover;
+		background-repeat: no-repeat;
+	}
+	.copperback,
+	.sustainability-text,
+	.grapeback,
+	.authenticity-text,
+	.education-text,
+	.tech-text {
+		margin-bottom: 0px;
+	}
+	/* Changing values card imgs to background imgs */
+	.values-card img {
+		display: none;
+	}
+	/* Individual card widths */
+	.sustainability-text {
```

Comment:
The same styles are being repeated for these cards. You could make a common class for these divs that wrapt the text 

- Change `<div class="authenticity-text">` -> `<div class="values-card-text">`

- In CSS: 
```
	 .values-card-text {
		display: flex;
		flex-direction: column;
		background-color: transparent;
		border-radius: 10px;
	}
	
.authenticity .values-card-text {   width: 50%; }

etc...
```

Reply:
We need to go over this please! We may need to keep some separated for reyse in other places.